app/org/sagebionetworks/bridge/services/AuthenticationServiceImpl.java

```diff
@@ -198,6 +198,7 @@ private UserSession getSessionFromAccount(Study study, Account account) {
         session.setAuthenticated(true);
         session.setEnvironment(config.getEnvironment().name().toLowerCase());
         session.setSessionToken(BridgeUtils.generateGuid());
+        session.setSessionId(BridgeUtils.generateGuid());
         session.setStudyIdentifier(study.getStudyIdentifier());
```

Comment:
See comment above. Is this for anonymity? If so could we call it internalSessionToken or something to queue later developers that it's not to be exposed to the response?